docroot/themes/custom/mass_theme/overrides/css/stacked-row-sections.css

```diff
@@ -0,0 +1,24 @@
+/**
+ * Align stat components in stacked row section (ex. org page) with other components.
+ */
+ .ma__stacked-row__section__stat-wrapper {
```

Comment:
Instead of adding an additional override class, I wonder if we can leverage the existing classes that I think @Javi-er added for aligning content in openmass `om__contained-component`